packages/vm/src/vm.ts

```diff
@@ -104,9 +104,16 @@ export class VM extends AsyncEventEmitter<VMEvents> {
 
     // TODO tests
     if (opts.eei) {
+      if (opts.evm) {
+        throw new Error('cannot specify EEI if EVM opt provided')
```

Comment:
Could you add a test for this case? And a test to show that VMs EEI is indeed the same EEI as the one used in EVM if one is passed on.

Reply:
Done!